core/store/migrate/migrations/0093_terra_txm.sql

```diff
@@ -0,0 +1,49 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE heads RENAME TO evm_heads;
+ALTER TABLE nodes RENAME TO evm_nodes;
+-- CREATE TABLE terra_chains (
+--     id text PRIMARY KEY,
+--     cfg jsonb NOT NULL DEFAULT '{}',
+--     created_at timestamptz NOT NULL,
+--     updated_at timestamptz NOT NULL
+-- );
+CREATE TABLE terra_nodes (
+    id serial PRIMARY KEY,
+    name varchar(255) NOT NULL CHECK (name != ''),
+    terra_chain_id text NOT NULL, -- REFERENCES terra_chains (id),
+	tendermint_url text CHECK (tendermint_url != ''),
+	fcd_url text CHECK (fcd_url != ''),
+	created_at timestamptz NOT NULL,
+	updated_at timestamptz NOT NULL
+);
+CREATE FUNCTION notify_terra_msg_insert() RETURNS trigger
+    LANGUAGE plpgsql
+AS $$
+BEGIN
+    PERFORM pg_notify('insert_on_terra_msg'::text, NOW()::text);
+    RETURN NULL;
+END
+$$;
+CREATE TABLE terra_msgs (
+	id BIGSERIAL PRIMARY KEY,
+-- TODO terra_chain_id text NOT NULL REFERENCES terra_chains (id),
+	contract_id text NOT NULL,
+    msg bytea NOT NULL,
+	state text NOT NULL,
```

Comment:
Probably want a couple of indexes here. `contract_id`, `state` and `tx_hash` most likely, plus BRIN indexes on created_at/updated_at are always handy and cheap.

Reply:
Might be nice to add a state machine CHECK constraint that requires presence of tx_hash in certain states also.

Reply:
tx_hash and contact_id wouldnt have any use yet, indexing the states would be useful but I'll wait until https://github.com/smartcontractkit/chainlink/pull/5814/files is merged